param/__init__.py

```diff
@@ -837,61 +827,56 @@ def crop_to_bounds(self,val):
 
         else:
             # non-numeric value sent in: reverts to default value
-            return  self.default
+            return self.default
 
         return val
 
+    def _validate_bounds(self, val, bounds, inclusive_bounds):
```

Comment:
I like this: more general than `_checkBounds` and makes the input explicit using arguments instead of `self`.

Reply:
Yeah, this is also in preparation to supporting validate methods on the class itself which could be useful for composite parameters.